apps/server/src/modules/account/services/account-idm.service.ts

```diff
@@ -160,15 +159,41 @@ export class AccountServiceIdm extends AbstractAccountService {
 	}
 
 	// eslint-disable-next-line @typescript-eslint/require-await, @typescript-eslint/no-unused-vars
-	async findMany(_offset: number, _limit: number): Promise<AccountDto[]> {
+	async findMany(_offset: number, _limit: number): Promise<Account[]> {
 		throw new NotImplementedException();
 	}
 
+	private async getOptionalIdmAccount(accountId: string): Promise<string | undefined> {
+		try {
+			return await this.getIdmAccountId(accountId);
```

Comment:
const xx = await this.getIdmAccountId(accountId)

return xxx;

or

return this.getIdmAccountId(accountId);

But both is not correct.